src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -3,7 +3,14 @@
 import java.util.Optional;
 import org.springframework.data.jpa.repository.JpaRepository;
 import wolox.training.models.Book;
+import wolox.training.models.User;
 
 public interface BookRepository extends JpaRepository<Book,Long> {
+
+    /**
+     *
```

Comment:
One of the most important things about javadocs is the explanation of the method being documented.
That description would be missing, indicating the purpose of the method in each of the documented repositories.